web/src/components/jobs/JobDetailPage.tsx

```diff
@@ -88,8 +86,9 @@ const JobDetailPage: FunctionComponent<IProps> = (props) => {
   }
   const i18next = require('i18next')
 
+  // return only latest row
   useEffect(() => {
-    fetchRuns(job.name, job.namespace)
+    fetchRuns(job.name, job.namespace, 10, 0)
     fetchJobTags(job.namespace, job.name)
   }, [job.name])
```

Comment:
Shouldn't we be setting this to a variable unless I'm missing something to page on this?

Reply:
It's because the "runs" are used by the job detail page so the thinking being that you pull the latest 10 runs so that job details works and then control all the paging from "Runs". Think there is a similar thing with "Dataset Versions" as that's pulled in the top level component and pushed downwards.